moduleroot/.travis.yml.erb

```diff
@@ -109,11 +109,31 @@ matrix:
       <%= key %>: <%= job[key].gsub(/@@SET@@/, set['set']).gsub(/@@COLLECTION@@/, set.fetch('collection', 'puppet6')).gsub(/@@TESTMODE@@/, set.fetch('testmode', 'apply')) %>
 <%   end -%>
 <% end -%>
+<% if @configs['use_litmus']
+  @configs['litmus']['puppet_collection'].each do |puppet_version|
+  @configs['litmus']['provision_list'].each do |platform|-%>
+    -
+<%  if @configs['litmus']['install_wget']-%>
+      before_script: ["bundle exec rake 'litmus:provision_list[<%=platform%>]'", "bundle exec bolt command run 'apt-get install wget -y' --inventoryfile inventory.yaml --nodes='localhost*'", "bundle exec rake 'litmus:install_agent[<%=puppet_version%>]'", "bundle exec rake litmus:install_module"]
```

Comment:
This will be problematic when the provision_list contains things which can and can't apt-install wget e.g. localhost windows

Reply:
This shouldn't really be an `if-else`.

psuedo code:
```
before_script  = ["bundle exec rake 'litmus:provision_list[<%=platform%>]'"]
if 'install_wget'
  before_script << "bundle exec bolt command run 'apt-get install wget -y' --inventoryfile inventory.yaml --nodes='localhost*'"
end
before_script << "next thing"
before_script << "next thing"
```

Reply:
Hey @glennsarti 
This is just a temporary solution. By default it is disabled. It will be fixed by this [ticket](https://tickets.puppetlabs.com/browse/MODULES-10067). 

Cheers!